dart_sp/9.Functions/11_New_methods_on_collections.md

```diff
@@ -15,7 +15,7 @@ void main() {
 
 La verdad solo se los mostré porque pueden verlo por ahí pero prueben escribir este ejemplo en Dartpad y vean qué sucede. 💀 Tómense su debido tiempo!
 
-_Avoid using `forEach` with a function literal._ Dice que evitemos utilizar el `forEach` en una función literal. Qué es aquello que le pasamos al `forEach`? Esta parte: `(person) => print(person)`? 💀 Nuevamente, piensen.
+_Function literals should not be passed to `forEach`._ Dice que evitemos utilizar el `forEach` en una función literal. Qué es aquello que le pasamos al `forEach`? Esta parte: `(person) => print(person)`? 💀 Nuevamente, piensen.
```

Comment:
@NahuelFerreroG Did you take a look at the error in the VSCode or Dartpad? Because this kind of errors I copyied directly from there.

Reply:
Yes that was I changed it, but i'll take a look again just now....
![image](https://user-images.githubusercontent.com/124050589/229759988-741cbb8d-d32f-437d-93ab-7739c31ccc0d.png)

Reply:
Perfect! Maybe they got updated!